packages/backend/src/managers/snippets/quarkus-snippet.ts

```diff
@@ -0,0 +1,26 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+import type { RequestOptions } from '@shared/src/models/RequestOptions';
+import mustache from 'mustache';
+import template from '../../templates/quarkus-langchain4j.mustache?raw';
+
+export function quarkusLangchain4Jgenerator(requestOptions: RequestOptions): string {
+  return mustache.render(template, {
+    baseUrl: requestOptions.url.substring(0, requestOptions.url.length - '/chat/completions'.length),
```

Comment:
`'/chat/completions'.length` might be extracted of the function as it's a constant but maybe the compiler already does that

Reply:
Done